src/main/kotlin/com/terraformation/backend/tracking/model/AssignedPlotDetails.kt

```diff
@@ -15,4 +16,33 @@ data class AssignedPlotDetails(
     val plantingSubzoneId: PlantingSubzoneId,
     val plantingSubzoneName: String,
     val plotName: String,
-)
+) {
+  fun gpxWaypoints(): List<GpxWaypoint> {
+    return listOf(
+        GpxWaypoint(
+            boundary.coordinates[SOUTHWEST].y,
+            boundary.coordinates[SOUTHWEST].x,
+            "$plotName Southwest Corner"),
+        GpxWaypoint(
+            boundary.coordinates[NORTHWEST].y,
+            boundary.coordinates[NORTHWEST].x,
+            "$plotName Northwest Corner"),
+        GpxWaypoint(
+            boundary.coordinates[NORTHEAST].y,
+            boundary.coordinates[NORTHEAST].x,
+            "$plotName Northeast Corner"),
+        GpxWaypoint(
+            boundary.coordinates[SOUTHEAST].y,
+            boundary.coordinates[SOUTHEAST].x,
+            "$plotName Southeast Corner"),
+    )
+  }
+
+  companion object {
+    // 0 Index vertices for monitoring plot corners.
+    const val SOUTHWEST = 0
+    const val NORTHWEST = 1
+    const val NORTHEAST = 2
+    const val SOUTHEAST = 3
```

Comment:
The polygons for monitoring plots don't have their vertices in this order -- they're ordered counterclockwise rather than clockwise. So it should be SW, SE, NE, NW. Obviously it's fine to export them in a different order in the GPX file if we want them to be clockwise there.